polygon-digital-carbon/schema.graphql

```diff
@@ -387,7 +387,7 @@ type ToucanBatch @entity {
 
 type C3RetireRequest @entity {
   "Request ID"
-  id: ID!
+  id: Bytes!
```

Comment:
is not having an `ID` type field on this entity going to create any side effects?

Reply:
It's only referenced internally so I don't see anywhere it could a side effect down the line. 

Redoing a full sync though, the last one didn't have the provenance record fix so it broke at that point. If all is fine on the sync then this can probably merge tomorrow

Reply:
It shouldn't. The request Id is a concatenation of the token address and retirement index so Bytes I think works here.